base/src/com/google/idea/blaze/base/sync/projectview/ImportRoots.java

```diff
@@ -101,8 +101,9 @@ public ImportRoots build() {
       if (buildSystem == BuildSystem.Bazel && hasWorkspaceRoot(rootDirectories)) {
         excludeBuildSystemArtifacts();
         excludeProjectDataSubDirectory();
-        excludeBazelIgnoredPaths();
       }
+      excludeBazelIgnoredPaths();
```

Comment:
Please don't move this out of the build system check. This should not run for Blaze.